modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,24 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, features are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+.Compatibility and support matrix
+[options="header"]
+|===
+
+| {clo} Version 6+| Component Version | OpenShift Version | Support Status
```

Comment:
Duplication of OpenShift version support creates a maintenance burden, link to the operator lifecycle page instead.

Reply:
Operator lifecycle page for future reference: https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic